README.md

```diff
@@ -135,7 +135,7 @@ please follow these instructions: https://developer.github.com/apps
 
 #### Install ngrok
```

Comment:
It could be worth adding a _bit_ of context on `ngrok` here before instructing people to install it "blind faith"... 🤔